access/src/main/external/static/templates/admin/changeTypeForm.html

```diff
@@ -0,0 +1,22 @@
+<form id="change_type_form" method="post" action="services/api/edit/changeType" class="edit_form change_type_form">
+	<p>
+		<div class="form_field">
+			<label>New Type</label>
+			<select id="change_type_new_type">
+				<option value="Aggregate" <%= "Aggregate Work" == defaultType? "disabled" : "" %>>Aggregate Work</option>
```

Comment:
I think all of the options should be available -- otherwise it's easy to accidentally change the type.

In that case, the label above should read "Type" rather than "New Type".